operator/pkg/central/values/translation/translation.go

```diff
@@ -70,8 +80,20 @@ func translate(c platform.Central) (chartutil.Values, error) {
 	if centralSpec == nil {
 		centralSpec = &platform.CentralComponentSpec{}
 	}
+	obsoletePVC := common.ObsoletePVC(c.GetAnnotations())
+	checker := &pvcStateChecker{
+		ctx:         ctx,
+		client:      t.client,
+		namespace:   c.GetNamespace(),
+		obsoletePvc: obsoletePVC,
```

Comment:
Can we just pass this into the GetCentralComponentValues instead of having it be a part of the pvcStateChecker?

Reply:
I think it is cleaner to group them in this instance and pass it all the way down. If there is no strong opinion, I will keep it this way.

Reply:
I won't block the PR because of it because the freeze is so close and the timezone difference increases communication latency, but I also think it would be beneficial to pass this bool separately. Making it a part of the checker seems to cause:
- only one dubious advantage of reducing `getCentralComponentValues` parameter list from 3 to 2.
- a disadvantage of the additional cognitive load on future readers to understand that this field does not affect the inner workings of this object.